test/test_with_xpress.jl

```diff
@@ -498,6 +498,25 @@ end
     @test results["Financial"]["lcc"] ≈ 6.71661825335e7 rtol=0.001
 end
 
+@testset "Outages with Wind and supply-to-load no greater than critical load" begin
+    input_data = JSON.parsefile("./scenarios/wind_outages.json")
+    s = Scenario(input_data)
+    inputs = REoptInputs(s)
+    m1 = Model(optimizer_with_attributes(Xpress.Optimizer, "MIPRELSTOP" => 0.01, "OUTPUTLOG" => 0))
+    m2 = Model(optimizer_with_attributes(Xpress.Optimizer, "MIPRELSTOP" => 0.01, "OUTPUTLOG" => 0))
+    results = run_reopt([m1,m2], inputs)
+
+    # Check that supply-to-load is equal to critical load during outages, including wind
+    supply_to_load = results["Outages"]["storage_discharge_series_kw"] .+ results["Outages"]["wind_to_load_series_kw"]
+    supply_to_load = [supply_to_load[:,:,i][1] for i in eachindex(supply_to_load)]
```

Comment:
Each index loops through all dims of supply_to_load, but you're using i in the last dim. Did this not throw an error?
Are you trying to test just the series corresponding t scenario 1, start time 1?

Reply:
There's only one outage start time and duration in the scenario.